pkg/k8shandler/visualization.go

```diff
@@ -473,11 +380,11 @@ func newKibanaPodSpec(cluster *logging.ClusterLogging, kibanaName string, elasti
 		"--upstream-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
 		"--https-address=:3000",
 		"-provider=openshift",
-		"-client-id=kibana-proxy",
-		"-client-secret-file=/secret/oauth-secret",
+		"-client-id=system:serviceaccount:openshift-logging:kibana",
+		"-client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token",
 		"-cookie-secret-file=/secret/session-secret",
 		"-upstream=http://localhost:5601",
-		"-scope=user:info user:check-access user:list-projects",
+		"-scope=user:info user:check-access",
```

Comment:
we don't need list-projects here?

Reply:
I'm uncertain why this is here as the Kibana proxy never needed to list projects.  All of that particular logic is in the openshift-elasticsearch-plugin.

Reply:
backtracking here as https://bugzilla.redhat.com/show_bug.cgi?id=1746695 was introduced.. not understood why our access script doesnt expose it